mammoth/models/model.py

```diff
@@ -58,24 +58,47 @@ def __init__(self, encoder, decoder, attention_bridge):
         self.decoder = decoder
         self.attention_bridge = attention_bridge
 
-    def forward(self, src, tgt, lengths, bptt=False, with_align=False, metadata=None):
-        dec_in = tgt[:-1]  # exclude last target from inputs
-
+    def forward(self, src, decoder_input, src_mask, return_attention=False, metadata=None):
         # Activate the correct pluggable embeddings and modules
-        self.encoder.activate(metadata)
-        self.decoder.activate(metadata)
-
-        enc_state, memory_bank, lengths, mask = self.encoder(src, lengths)
-
-        memory_bank, alphas = self.attention_bridge(memory_bank, mask)
+        active_encoder = self.encoder.activate(
+            task_id=metadata.corpus_id,
+            adapter_ids=metadata.encoder_adapter_ids,
+        )
+        active_decoder = self.decoder.activate(
+            task_id=metadata.corpus_id,
+            adapter_ids=metadata.decoder_adapter_ids,
+        )
+
+        # # QUI logging for batch shapes
```

Comment:
is there any reason to keep this commented block?

Reply:
I'll remove it.